src/windows.rs

```diff
@@ -84,23 +86,33 @@ pub fn statvfs(path: &Path) -> Result<FsStats> {
     let root_path: &mut [u16] = &mut [0; 261];
     volume_path(path, root_path)?;
     unsafe {
+        let mut free_space = 0;
+        let mut total_space = 0;
+        let ret = GetDiskFreeSpaceExW(
+            root_path.as_ptr(),
+            &mut free_space,
+            &mut total_space,
+            std::ptr::null_mut(),
+        );
+        if ret == 0 {
+            return Err(Error::last_os_error());
+        }
+
         let mut sectors_per_cluster = 0;
         let mut bytes_per_sector = 0;
-        let mut number_of_free_clusters = 0;
-        let mut total_number_of_clusters = 0;
+        let mut _number_of_free_clusters = 0;
```

Comment:
Why variable starting with underscore? It will be silences when not used, which is not a good thing.